src/theme/book.js

```diff
@@ -521,10 +520,13 @@ function playpen_text(playpen) {
     }, { passive: true });
 
     // Scroll sidebar to current active section
-    var activeSection = sidebar.querySelector(".active");
-    if (activeSection) {
-        sidebarScrollBox.scrollTop = activeSection.offsetTop;
-    }
+    window.addEventListener('load', function() {
```

Comment:
Is it necessary to add an event listener here? This script is loaded at the end of `index.hbs` after all the HTML for the sidebar, so that's why the existing code was able to directly query for the `.active` element.

Reply:
I'm just too careful. Just querying the `.active` element is not enough, maybe at the end there are some CSS or JS change the width or height of elements dramatically, which would break the scrolling. So I wait until the page totally rendered, that is window loaded.

Reply:
It's a good instinct to be careful, however the `load` event doesn't guarantee that JavaScript has finished manipulating the DOM. It just means that HTML and dependent resources (such as images and stylesheets) have been loaded: https://developer.mozilla.org/en-US/docs/Web/API/Window/load_event. Given that there is other code in this file which assumes that the DOM has stabilised, I think this event listener is a bit unnecessary. Of course it's not harmful though either (unless some images take a long time to load).

Reply:
The addition of this event listener causes significant jumping and flickering on chrome and safari browsers.  Would it be possible to remove it?